Python/getargs.c

```diff
@@ -2027,14 +2022,35 @@ parser_init(struct _PyArg_Parser *parser)
     parser->min = min;
     parser->max = max;
     parser->kwtuple = kwtuple;
-    parser->initialized = owned ? 1 : -1;
-
     assert(parser->next == NULL);
     parser->next = static_arg_parsers;
     static_arg_parsers = parser;
+    parser->initialized = owned ? 1 : -1;
```

Comment:
Why move this line (and remove the blank line)?  It was a bit more clear the way it was.

Reply:
done just pushed the fix